roles-in-tech/security.md

```diff
@@ -1,71 +1,114 @@
 # Security
 
+![security locks](\roles-in-tech\security-locks.jpg)
+
 ### Projected Time
 90-180 minutes
 
 ### Prerequisites
-How the Internet Works Topic Outline
+- To become security engineer, it is recommended to  pursue bachelors degree in computer or have experience in security related field. Because employers prefer those candidates.
+- [How the Internet Works Topic Outline](https://github.com/Techtonica/curriculum/blob/master/requests-and-responses/requests-and-responses.md)
+- [Intro to Security](https://github.com/Techtonica/curriculum/blob/master/security/web-security.md)
 
 ### Motivation
+- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly.
+
+- Security is the process of **preventing** and **detecting** unauthorized use of computer. *Prevention* helps in stopping unauthorized users to access part of the system. *Detection* helps to determine whether or not someone has tried to break into your system.
+
 Apprentices will learn secure development basics, common pitfalls, and how to avoid them.
 
 ### Objectives
-**Participants will be able to:**
-- Pull a relevant JS library up to handle common scenarios
-- Validate user input
-- Authenticate users on a site
-- XSS someone else's web page
+**Participants will be able to learn:**
+- Risk Management
+- Cryptography & PKI
+- Identity and access Management
+- How to implement secure n/w architecture and system design
+- Discover security vulnerabilities across an entire network
 
 ### Specific Things To Teach
-- OWASP Secure coding practices
-	- Input validation
-	- Authentication means and pitfalls
-	- Session management
-	- Cross-site scripting (XSS)
-	- Cross-site request forgery (CSRF)
+The basic required skills as an Security Engineer are
+- Programming Skills
+- Communication Skills
+- Risk Management
+- Networking Basics
+- Situational Awareness
+- Toolkit Maintenance
+
+One can join course [CompTIA Security+ Certification](https://certification.comptia.org/certifications/security) or [CCNA Security](https://www.cisco.com/c/en/us/training-events/training-certifications/certifications/associate/ccna-security.html#~stickynav=1) to improve the basics of security
+
+After gaining basic skills one can go further as *Security Architect* or *Security Analyst* or *CISO* or any further specialization
 
 ### Materials
 
 - [Techtonica's Roles in Tech Presentation](https://drive.google.com/file/d/1oXgOX1EC4igI6ZwgJlhF6AN4dG2Yn_S4/view) (security section starts at 2:57)
-- [A quick introduction to web security [FreeCodeCamp]](https://medium.freecodecamp.org/a-quick-introduction-to-web-security-f90beaf4dd41)
+- online course at [CYBRAY](https://www.cybrary.it/)
```

Comment:
Great resources, but are you assigning all of these to be done during the 30-45 window?  If they're not assigned, maybe you can put them in a new category like you have for "conferences" and "organizations."